.github/workflows/testnet.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           VITE_PAIR_CONTRACT: "0xaE42EA9864C36820627068a11959915f817ce809"
           VITE_MOM_CONTRACT: "0x2cFdb89954991b69D6D4B2ec8b70f3b995F4E8Ec"
           VITE_TRADER_CONTRACT: "0xbE005Fd6328960413f2627Ebb489B09507B2AEdF"
+          VITE_FROG_CONTRACT: "0x52A5ddFF59E0A27a1cBA75c8045A434b7D296A6c"
```

Comment:
Is the testnet address meant to be the same as the mainnet address?

Reply:
Yes, they're the same contract:

Testnet address:
https://mumbai.polygonscan.com/address/0x52a5ddff59e0a27a1cba75c8045a434b7d296a6c

Mainnet address:
https://polygonscan.com/address/0x52a5ddff59e0a27a1cba75c8045a434b7d296a6c

I didn't change anything from the code on the testnet, I think that's why it generated the same address 😅